api/queries.py

```diff
@@ -57,154 +33,50 @@
-        resolver=resolve_get_group_by_group,
-        description="Allows the selection of group information from a given group enum"
-    )
-    get_group_by_sector = graphene.List(
-        of_type=Groups,
-        sector=graphene.Argument(SectorEnums, required=True),
-        resolver=resolve_get_group_by_sector,
-        description="Allows selection of groups from a given sector enum"
-    )
-    get_org_by_id = graphene.List(
-        of_type=Organizations,
-        id=graphene.Argument(graphene.Int, required=True),
-        resolver=resolve_get_org_by_id,
-        description="Allows the selection of an organization from a given ID"
-    )
-    get_org_by_org = graphene.List(
-        of_type=Organizations,
-        org=graphene.Argument(OrganizationsEnum, required=True),
-        resolver=resolve_get_org_by_org,
-        description="Allows the selection of an organization from its given organization code"
-    )
-    get_org_by_group = graphene.List(
-        of_type=Organizations,
-        group=graphene.Argument(GroupEnums, required=True),
-        resolver=resolve_get_orgs_by_group,
-        description="Allows the selection of organizations from a given group"
-    )
-    get_domain_by_id = graphene.List(
-        of_type=Domains,
-        id=graphene.Argument(graphene.Int, required=True),
-        resolver=resolve_get_domain_by_id,
-        description="Allows the selection of a domain from a given ID"
-    )
-    get_domain_by_domain = graphene.List(
-        of_type=Domains,
-        url=graphene.Argument(URL, required=True),
-        resolver=resolve_get_domain_by_domain,
-        description="Allows the selection of a domain from a given domain"
-    )
-    get_domain_by_organization = graphene.List(
-        of_type=Domains,
-        org=graphene.Argument(OrganizationsEnum, required=True),
-        resolver=resolve_get_domain_by_organization,
-        description="Allows the selection of domains under an organization"
-    )
-    get_scan_by_id = graphene.List(
-        of_type=Scans,
-        id=graphene.Argument(graphene.Int, required=True),
-        resolver=resolve_get_scan_by_id,
-        description="Allows the selection of a scan from a given scan ID"
-    )
-    get_scans_by_date = graphene.List(
-        of_type=Scans,
-        date=graphene.Argument(graphene.Date, required=True),
-        resolver=resolve_get_scans_by_date,
-        description="Allows selection of scans on a given date"
-    )
-    get_scans_by_date_range = graphene.List(
-        of_type=Scans,
-        startDate=graphene.Argument(graphene.Date, required=True),
-        endDate=graphene.Argument(graphene.Date, required=True),
-        resolver=resolve_get_scans_by_date_range,
-        description="Allows selection of scans from a given date range"
-    )
-    get_scans_by_domain = graphene.List(
-        of_type=Scans,
+    users = SQLAlchemyConnectionField(UserObject._meta.connection, sort=None)
```

Comment:
`all_users` describes what this does more than `users`.  If the change was requested then its no problem!  Just my $0.02!

Reply:
As it is selecting the users table I felt like it was a more appropriate name, I believe it will also be changed up in the future so it is more of a place holder for testing user functionality currently